src/utils/accounts.js

```diff
@@ -17,67 +17,82 @@
     with this program; if not, write to the Free Software Foundation, Inc.,
     51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 */
-import get from 'lodash.get';
 import type { Account, Accounts, AccountTypes } from 'models/Account';
 import { ACCOUNT_TYPES } from 'constants/accountsConstants';
 import { addressesEqual } from './assets';
 
-export function getActiveAccount(accounts: Accounts = []): ?Account {
+export const getActiveAccount = (accounts: Accounts): ?Account => {
```

Comment:
What if just `...  => accounts.find(({ isActive }) => isActive);`?

Reply:
I don't think that's gonna improve readability :thinking: